docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+
+
+Manual Updates
+--------------
+
+However, because the *Secure Viewing Station (SVS)* is airgapped, it cannot
+receive upgrade notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the SecureDrop Application private key), so we **strongly** encourage you to manually upgrade the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 weeks: https://tails.boum.org/support/faq/index.en.html
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Tails Manual Update Process at a Glance
+----------------------------------------
+
+The process for manually updating a Tails workstation is:
```

Comment:
The document sort of switches back and forth between trying to be generic vs. being specific to the SVS. 

I'd recommend a scrub to either make this just about the SVS (and then treat the other case in a separate section), or to more consistently refer to "the USB you want to upgrade".

Reply:
Good call. Switched to "SVS" throughout (for clarity, to avoid having yet another long descriptor name for a USB--"the USB you want to back up", "the primary USB" etc all gets a bit confusing). Left the note at the top of the instructions that this advice would work equally on any Tails USB.